doc/python/simulation.rst

```diff
@@ -114,34 +114,34 @@ over the local and distributed hardware resources (see :ref:`pydomdec`). Then, t
 
     **Sampling probes:**
 
-    .. function:: sample(probe_id, schedule, policy)
+    .. function:: sample(probeset_id, schedule, policy)
 
-        Set up a sampling schedule for the probes associated with the supplied probe_id of type :py:class:`cell_member`.
+        Set up a sampling schedule for the probes associated with the supplied probeset_id of type :py:class:`cell_member`.
         The schedule is any schedule object, as might be used with an event generator — see :ref:`pyrecipe` for details.
         The policy is of type :py:class:`sampling_policy`. It can be omitted, in which case the sampling will accord with the
         ``sampling_policy.lax`` policy.
 
-        The method returns a handle which can be used in turn to retrieve the sampled data from the simulator or to
+        The method returns a :term:`handle` which can be used in turn to retrieve the sampled data from the simulator or to
         remove the corresponding sampling process.
 
-    .. function:: probe_metadata(probe_id)
+    .. function:: probe_metadata(probeset_id)
 
-        Retrieve probe metadata for the probes associated with the given probe_id of type :py:class:`cell_member`.
+        Retrieve probe metadata for the probes associated with the given probeset_id of type :py:class:`cell_member`.
         The result will be a list, with one entry per probe; the specifics of each metadata entry will depend upon
         the kind of probe in question.
 
     .. function:: remove_sampler(handle)
 
-        Disable the sampling process referenced by the argument ``handle`` and remove any associated recorded data.
+        Disable the sampling process referenced by the argument :term:`handle` and remove any associated recorded data.
 
     .. function:: remove_all_samplers()
 
         Disable all sampling processes and remove any associated recorded data.
 
     .. function:: samples(handle)
 
-        Retrieve a list of sample data associated with the given ``handle``.
-        There will be one entry in the list per probe associated with the :term:`probe id` used when the sampling was set up.
+        Retrieve a list of sample data associated with the given :term:`handle`.
+        There will be one entry in the list per probe associated with the :term:`probeset id` used when the sampling was set up.
```

Comment:
If we're doing ```:term:`probeset_id` ```then there's a lot of missed occurences that are just `probeset_id`